modal/_partial_function.py

```diff
@@ -722,3 +726,33 @@ def wrapper(raw_f: Callable[..., Any]) -> _PartialFunction:
         )
 
     return wrapper
+
+
+def _concurrent(
+    _warn_parentheses_missing=None,
+    *,
+    max_inputs: int,  # Limit on each contianer's concurrency
```

Comment:
Comment typo: `contianer`

I know this is beating a dead horse but something about the name `max_inputs` still itches for me (not a blocker though, I think it's more important to just get the changes out there).

Did we ever consider `max_concurrency` as the public name for this attribute - I still have a feeling `max_inputs` could intuitively indicate a cap on total the number of inputs?

Reply:
We considered a lot of things :) https://www.notion.so/modal-com/The-modal-concurrent-decorator-1ad1e7f169498050974ac8a1cd3a2bec?pvs=4

Reply:
I know, but I couldn't find `max_concurrency` in that doc 😬

Reply:
It's equivalent to the `limit` proposal IMO.